Microsoft.VisualStudio.CodeTools/PropertyPage/Common.cs

```diff
@@ -52,7 +52,7 @@ static public String GetLocalRegistryRoot()
             // Get the visual studio root key
             if (VsRoot == null)
             {
-                ILocalRegistry2 localReg = GetService(typeof(ILocalRegistry)) as ILocalRegistry2;
+                ILocalRegistry2 localReg = GetService(typeof(SLocalRegistry)) as ILocalRegistry2;
```

Comment:
From the [documentation for `ILocalRegistry2`](https://msdn.microsoft.com/en-us/library/microsoft.visualstudio.shell.interop.ilocalregistry2.aspx):

> **Notes to Callers**
> Called by any package that wants to register in the Visual Studio section of the registry. This interface is obtained by using the [`SLocalRegistry`](https://msdn.microsoft.com/en-us/library/microsoft.visualstudio.shell.interop.slocalregistry.aspx) service.